snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-0

```diff
@@ -35,6 +35,32 @@
         ]
       }
     },
+    "metrics": {
+      "description": "Key-value pairs of metrics",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metric attribute",
+        "properties": {
```

Comment:
**question:** 

Since I cannot find it yet: How did we decide about the `tags` property which was listed [here](https://www.notion.so/metabase/Migrate-Anonymous-Stats-ping-to-Snowplow-18f155e235d34ba09f45518a866336e2?pvs=4#e3d7fdc32d3446df9e486ce8d63da04b)?

Reply:
Thanks for the pointer. I don't see any tags in the product doc, but I will add support for them to the schema and stats collecting code.

Reply:
Done.

Reply:
Thanks for adding!
Yeah I think there have been many docs in the meanwhile. FYI [This](https://www.notion.so/metabase/Migrate-Anonymous-Stats-ping-to-Snowplow-18f155e235d34ba09f45518a866336e2?pvs=4#e3d7fdc32d3446df9e486ce8d63da04b) was the one I was referring to.